components/SingleArticlePage.jsx

```diff
@@ -77,7 +77,7 @@ export default function SingleArticlePage() {
       </p>
 
       <h3>Comments:</h3>
-      <CommentCard article_id={article_id} />
+      <CommentCard article_id={article_id} username={username} />
```

Comment:
You could put the username in a context. save you having to pass props everywhere